.github/workflows/sonar.yaml

```diff
@@ -1,24 +1,67 @@
 name: Sonar
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
Should this be triggered by pushes to the `development` branch instead? `development` is ultimately what ends up going to production, as we don't use a `master` or `main` branch.

Reply:
Yes. Good catch.